compute_runtime/doc.go

```diff
@@ -0,0 +1,5 @@
+// Copyright 2024 Fastly, Inc.
+//
+// Package compute_runtime provides ways to reflect on the operation of the
+// compute runtime in which your application is operating.
```

Comment:
I know this is weird but here and in other places you'll want to separate the copyright header comment from the comment block that applies to the `package` line or the `package` line itself.  Otherwise it will end up in the documentation.

Reply:
Cool, think I caught these in ff3c1f5